client_linux_integration_test.go

```diff
@@ -45,21 +45,21 @@ func TestIntegrationLinuxConcurrent(t *testing.T) {
 	defer wg.Wait()
 
 	for i := 0; i < workers; i++ {
-		go func() {
+		go func(differentI int) {
```

Comment:
I guess this improvement is for debugging tests and is unrelated to the actual feature. Usually I'd like to do this as a separate change. But this is simple enough, and thanks for documenting it in the PR description.

Reply:
Yeah, I didn't really feel completely comfortable leaving this in there. 
But it only changes cosmetics on a failure of the load test. It doesn't change the direct behavior and outputs of the successful tests. 
And it was helpful while testing on rather slow devices.